src/Senseval3.jl

```diff
@@ -0,0 +1,82 @@
+struct Senseval3{S}
+    filepaths::Vector{S}
+end
+
+function Senseval3(dirpath, ver=3.0)
+    @assert(isdir(dirpath), dirpath)
+
+    if "wordnet$ver" ∈ readdir(dirpath)
+        paths = joinpath.(dirpath,"wordnet$ver", readdir(joinpath(dirpath, "wordnet$ver")))
+    else
+        paths = joinpath.(dirpath, readdir(dirpath))
+    end
+    Senseval3(paths)
+end
+
+Senseval3() = Senseval3(datadep"Senseval 3")
+
+MultiResolutionIterators.levelname_map(::Type{Senseval3}) = [
+    :doc=>1, :contextfile=>1, :context=>1,
+    :sent=>2,
+    :word=>3, :token=>4,
+    :char=>5]
+
+
+function parse_sense_annotated_word_senseval3(line::AbstractString)
+    captures = match(r"<wf cmd=done.* pos=(.*) lemma=(.*) wnsn=(.*) lexsn=(\d.*:\d*|[U]|NOANSWER|NOWN).*>(.*).*</wf>", line).captures
+    pos, lemma, wnsn, lexsn, word = captures
+    if ';' in wnsn
+        # Discard Extra Senses
+        wnsn = split(wnsn, ';') |> first
+        lexsn = split(lexsn, ';') |> first
+    end
+
+    SenseAnnotatedWord(pos, lemma, parse(Int, wnsn), lexsn, word)
+end
+
+
+function parse_senseval3file(filename)
+    local sent
+    lines = @NestedVector(TaggedWord,2)()
+    context = Document(intern(basename(filename)), lines)
+
+    ignore(line) = nothing
```

Comment:
move `ignore` to the `apply_subparsers` file,
and document it as a helper for ignoring lines when using subparsers.
Also delete it from semcor